app/models/information.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: information
+#
+#  id    :bigint           not null, primary key
+#  text  :text
+#  title :integer          not null
+#
+# Indexes
+#
+#  index_information_on_title  (title) UNIQUE
+#
+
+class Information < ApplicationRecord
+  enum title: { about: 0 }
```

Comment:
It's weird that you have an enum but only one option 🤔

Reply:
Yeah, this is supposed to grow when new information sections are included. This was @JuanfraM idea

Reply:
@t-romani  @gonzaM the idea here was to have control about the differents possible titles instead of having just a string. Later title could be:
`enum title: { about: 0, phone_number: 1, address: 2 }`
Just that.